lib/ansible/modules/cloud/amazon/iam_cert.py

```diff
@@ -50,19 +50,19 @@
     default: "/"
   cert_chain:
     description:
-      - The CA certificate chain in PEM encoded format.
-      - Note that prior to 2.4, this parameter expected a path to a file.
-        Since 2.4 this is now accomplished using a lookup plugin. See examples for detail.
+      - The path to the CA certificate chain in PEM encoded format.
+        As of 2.4 this can also be accomplished using a lookup plugin.
```

Comment:
This comment was because the module went from supporting only paths to only supporting content.
I'm fine changing this if it doesn't sense, but maybe you can phrase that for me? I feel like this is already essentially saying that as of 2.4 content and paths are permitted. "... can also be accomplished using a lookup plugin."